src/components/dialogs/field-hook.js

```diff
@@ -137,6 +141,9 @@ export const useFileValue = ({ fileExceedsLimitMessage, isLoading }) => {
             setFileOk(false);
         }
     }, [selectedFile, fileExceedsLimitMessage, intl]);
+    if (noSetFileNeeded) {
```

Comment:
Don't understand

Reply:
I used it in order to return specific fields. But as discussed there is another way to do it.